README.md

```diff
@@ -25,7 +25,9 @@ states and produce helpful logs.
 
 Install and run Matrix by doing the following:
 
-    sudo pip3 install 'git+https://github.com/juju-solutions/matrix.git'
+    git clone https://github.com/juju-solutions/matrix.git
+    cd matrix
+    sudo pip3 install . -f wheelhouse
```

Comment:
zookeeper.h sounds like its pulling the old python (pre-juju 1.0) bindings that still are in pip and called juju. If you have that sitting around somewhere it would try to build the zookeeper bindings, but nothing modern uses it